### PR TITLE
Fix wrong wait condition

### PR DESCRIPTION
**Pull request by HeavyWombat**
The `wait` command stopped waiting when there was pod in the namespace.

Change logic to never return when there are no pods in the namespace.

### CODE REVIEW

File: internal/cmd/wait.go [hunk 1]
```diff
@@ -134,7 +134,7 @@ whether all pods that match the configured prefix to become ready.`,
 				}
 
 				if len(list) == 0 {
-					return true, nil
+					return false, nil
```
Comment (qu1queee) [inline internal/cmd/wait.go]: if the `checkFunc` will always return false, do you need to return `boolean`, shouldn´t this be returning only an `error` ?

Comment (HeavyWombat) [reply internal/cmd/wait.go]: Thanks for spotting this. I will have a look.

Comment (HeavyWombat) [reply internal/cmd/wait.go]: The condition did not take into account that there should be pods that match the condition and all of them have to report ready status.

**Comment by HeavyWombat** [comment]
@qu1queee If you have time, would be nice to check the PR again. I fixed the condition that was wrong an added test cases around it.

**Comment by qu1queee** [comment]
I will take a look asap

**Pull request merged**